members/B000944.yaml

```diff
@@ -11,10 +11,14 @@ contact_form:
         selector: "#fname"
         value: "$NAME_FIRST"
         required: true
+        options:
+          blacklist: "'"
```

Comment:
do you need to escape this?

Reply:
I don't think so. Did a few tests and msgs went thru succesfully